MUSTsegmenter/MUSTsegmenter.py

```diff
@@ -1334,7 +1330,7 @@ def loadTestData(self):
     for r in roiNames:
       rObj = open(os.path.join(sampleDataPath, r + '.json'))
       coords = json.load(rObj)
-      roi = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLAnnotationROINode")
```

Comment:
The Annotations module was officially retired and removed yesterday. https://github.com/Slicer/Slicer/commit/f44849b014ecf9a245901239db64037556110970